shell/pages/c/_cluster/fleet/index.vue

```diff
@@ -452,26 +452,28 @@ export default {
         </h1>
 
         <div class="dashboard-main-actions">
-          <div :data-testid="'fleet-dashboard-expand-all'">
-            <p
-              v-if="allCardsExpanded"
-              @click="toggleCardAll('collapse')"
-            >
-              {{ t('fleet.dashboard.collapseAll') }}
-            </p>
-            <p
-              v-else
-              @click="toggleCardAll('expand')"
-            >
-              {{ t('fleet.dashboard.expandAll') }}
-            </p>
-          </div>
           <ButtonGroup
             :data-testid="'view-button'"
             :value="viewMode"
             :options="viewModeOptions"
             @update:value="viewMode = $event"
           />
+          <RcButton
+            small
+            ghost
+            data-testid="fleet-dashboard-expand-all"
+            @click="toggleCardAll(allCardsExpanded ? 'collapse' : 'expand')"
+          >
```

Comment:
nit: I feel like this button would benefit from having a fixed width so that the content doesn't shift left and right when toggling between expand/collapse. Perhaps, sync with UX to see if this makes sense.